smarts/sstudio/genhistories.py

```diff
@@ -487,18 +527,7 @@ def read_dataset(path: str) -> Generator[bytes, None, None]:
                 _ = f.read(4)  # masked_crc32_of_data (ignore)
                 yield record_data
 
-    @property
-    def rows(self) -> Generator[Dict, None, None]:
-        def lerp(a, b, t):
-            return t * (b - a) + a
-
-        def constrain_angle(angle):
-            """Constrain to [-pi, pi]"""
-            angle = angle % (2 * math.pi)
-            if angle > math.pi:
-                angle -= 2 * math.pi
-            return angle
-
+    def _get_scenario(self):
```

Comment:
Missing return type hint.

Reply:
It was removed because its return type is `scenario_pb2.Scenario` which when shown as a return type hint throws a `NotImplementedError` if the waymo dependencies fail to install above in the import try block.